jupyterlab_bigquery/jupyterlab_bigquery/handlers.py

```diff
@@ -89,13 +89,25 @@ def get_dataset_details(client, dataset_id):
       }
   }
 
-def get_schema(schema):
-    return [{
+def format_detail_schema(schema):
+  formatted_schema = []
+  for field in schema:
+    formatted_schema.append({
         'name': field.name,
         'type': field.field_type,
         'description': field.description,
         'mode': field.mode
-    } for field in schema]
+      })
+    if field.field_type == 'RECORD':
+      for record_entry in field.fields:
+        formatted_schema.append({
+          'name': "{}.{}".format(field.name, record_entry.name),
+          'type': record_entry.field_type,
+          'description': record_entry.description,
+          'mode': record_entry.mode
+        })
```

Comment:
Handles records, but only singly-nested records for now. A future PR will address deeply nested records.